README.md

```diff
@@ -126,153 +133,189 @@ editorElement.initEditor(arrayOfPluginNames, userConfigObject);
+- `setHtmlContent(content: string)`: set the HTML content inside the editor, overwriting all previous content.  
+- `getHtmlContent()`: Get the HTML content of the editor. This can be different than custom content set via `setHtmlContent`, because of HTML parsing logic.
+
+#### controller API
+These methods are accessible via `editorElement.controller` and contain a way to directly interact with the Prosemirror logic underneath. This is an instance of [SayController](https://github.com/lblod/ember-rdfa-editor/blob/d4472d2e237256d30333cfcc20ce6eea7db241f2/addon/core/say-controller.ts). Not all possible methods are shown.
+- `focus()`: focus the window to the main editor view
+- `setHtmlContent(content: string)`: sets the content of the main editor, overwriting all previous content.
+- `htmlContent`: property containing the "serialized" html content of the editor. This is essentially the raw content without all the plugin bells and whistles, suitable for storing in a database. It can then be loaded with `setHtmlContent`.
+- `doCommand(command: Command, { view = this.activeEditorView } = {})`: executes a [Prosemirror command](https://prosemirror.net/docs/guide/#commands) on the main view. A different view can be provided, which is mainly used internally to control nested editor instances (e.g. for the implementation of the variables)
+- `checkCommand(command: Command, { view = this.activeEditorView } = {})`: checks whether a [Prosemirror command](https://prosemirror.net/docs/guide/#commands) may be executed. Often used together with `doCommand` to disable an action if it is not allowed.
+- `isMarkActive(mark: MarkType)`: checks whether a mark is currently active. This is currently of not much use in this package, since we do not expose the MarkType interface yet. (But for the curious, this is what the toolbar buttons use to update their active state).
+- `withTransaction(callback: (tr: Transaction) => Transaction | null, includeEmbeddedView = false)`: apply a [Prosemirror transaction](https://prosemirror.net/docs/ref/#state.Transaction) on the main view (or currently active embedded view). When you want to apply the transaction, the callback should return the transaction object.
+- `mainEditorState`: the [editor state](https://prosemirror.net/docs/ref/#state.Editor_State) instance of the main editor
+- `activeEditorState`: the state instance of the active editor. This is usually the same as the `mainEditorState`, except when inside a nested instance (like in a variable field). For typical use, the `mainEditorState` is the one you will most likely need.
+- `mainEditorView`: the [view](https://prosemirror.net/docs/ref/#view.EditorView) instance of the main editor
+- `activeEditorView`: the view instance of the active editor. (see above for the distinction between the main and active editor)
+- `setActiveView(view: RdfaEditorView)`: activate a specific view.
```

Comment:
I'm not sure if this method needs to be documented for embeddable, as it has quite a specific use case for embedded editors.

Reply:
I think the reason is that if they want to use custom commands, it might work differently/weirdly for variables and such. But I am also not sure if there are many cases where a command they'll write will really need to know the difference.
Maybe leaving it out but instead add a note at the bottom about it is a better idea?

Reply:
Yes, maybe or in a section on custom prosemirror commands.

Reply:
@elpoelma I added a separate section with some explanation about prosemirror commands, linking to the editor wiki.
The explanation of what the difference is between the main and active editor state should really be part of that wiki instead of trying to explain it here without (the wiki has context to make the explanation easier).

If you could check the new addition if it is okay, that would be great :)

Reply:
looks great!